docker-compose.yml

```diff
@@ -9,21 +9,32 @@ services:
       - weblate-data:/app/data
     env_file:
       - ./environment
+    ports:
+      - "8000:8000"
     restart: always
     depends_on:
       - database
       - cache
   web:
-    build: nginx
-    links:
-      - weblate
+    image: jwilder/nginx-proxy
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - ./certs:/etc/nginx/certs:ro
+      - /etc/nginx/vhost.d
+      - /usr/share/nginx/html
     ports:
       - "80:80"
-    volumes_from:
-      - weblate:ro
```

Comment:
So the plan would be to first implement #50 and then add letsencrypt support directly in the weblate container to avoid the performance penalty of the reverse proxy. If that's where you want to go, I'll work in this direction to make it happen.